test/unit/query-interface/bulk-insert.test.ts

```diff
@@ -12,6 +13,22 @@ describe('QueryInterface#bulkInsert', () => {
     sinon.restore();
   });
 
+  it('uses minimal insert queries', async () => {
```

Comment:
The test I had in mind was more an integration test where you try to bulkInsert 1001 rows, then count the results, to assert that it does work :)

Reply:
Hello @ephys,

As @WikiRik mentioned above, there is already an integration test that is testing the bulkInsert query.
Does this test suffice?

The integration test to insert 1000 or 1001 rows should always pass as the previous code could complete the bulkInsert, the issue is that it uses multiple insert statements.

This is why I went with the unit test instead as this way I was able to ensure that only one insert statement is made.